packer/jenkins-agent-win2019-x64-gradle-check.json

```diff
@@ -87,7 +87,12 @@
     {
       "type":"powershell",
       "inline": [
-        "C:\\ProgramData\\Amazon\\EC2-Windows\\Launch\\Scripts\\InitializeInstance.ps1 -Schedule",
+        "C:\\ProgramData\\Amazon\\EC2-Windows\\Launch\\Scripts\\InitializeInstance.ps1 -Schedule"
```

Comment:
Note: so we split running these two scripts to make sure we are running one by one instead of together to avoid race condition?

Reply:
Yes, seems like right now the second script run success will overshadow the failure of the first script at some point.
So just separate them out so we will have a consistent outcome.

Thanks.